test/stmt/errors_nonobjc.swift

```diff
@@ -1,18 +0,0 @@
-// RUN: %empty-directory(%t)
```

Comment:
Similarly I think it makes sense to keep this test and update the comment and expected output.

Reply:
Similarly, ok :)